### DISCUSSION TITLE
Which is the correct way to do subsequent deployments?

### DISCUSSION THREAD

**Discussion by efonsecab** [microsoft/aspire#4433] [Q&A]
So, I have a system I deployed into Azure using azd up.
I did however, apply custom domains from within the portal, additionally I made several more changes to the app and re-deploy using azd up.
Once I re-deployed, my Custom Domains were reset.
Which should be the right way to avoid having to re-configure the Custom Domains each time I deploy updates of my app?

**Comment by davidfowl** [comment]
See https://github.com/dotnet/aspire/issues/1498
